bisq-easy/src/main/java/bisq/bisq_easy/BisqEasyNotificationsService.java

```diff
@@ -89,17 +87,15 @@ private void handleNotification(ChatNotification notification) {
             return;
         }
 
-        tradeIdsOfNotifications.setAll(chatNotificationService.getNotConsumedNotifications(ChatChannelDomain.BISQ_EASY_OPEN_TRADES)
-                .flatMap(chatNotification -> chatNotification.getTradeId().stream())
+        tradeNotifications.setAll(chatNotificationService.getNotConsumedNotifications(ChatChannelDomain.BISQ_EASY_OPEN_TRADES)
+                .filter(chatNotification -> !dismissedNotifications.contains(chatNotification))
                 .collect(Collectors.toSet()));
-        // Reset dismissed state if we get a new notification
-        if (!tradeIdsOfNotifications.isEmpty()) {
-            isNotificationPanelDismissed.set(false);
-        }
         updateNotificationVisibilityState();
     }
 
     public CompletableFuture<Boolean> shutdown() {
+        chatNotificationService.getChangedNotification().removeObserver(this::handleNotification);
```

Comment:
We use the result from `chatNotificationService.getChangedNotification().addObserver` to unbind...